code/modules/codex/entries/faction_codex.dm

```diff
@@ -0,0 +1,87 @@
+/datum/codex_entry/TG_faction
+    display_name = "Terran Government (faction)"
+    lore_text = "The Terran Government, known informally as the TG, is a militaristic type of democracy, requiring military service and bolstering a \
+    a large, generally capable navy.<br> The Terran Government traces back to the Intercelestial Space Authority, an organization created to legislate \
+    and enforce Space Law, which over time amassed more influence until eventually managing to absorb most of the UN, leading to the modern-day government. \
+    The Terran Government is the largest human authority in space, currently fighting a war against the ICC, one that has however been put on delay due to \
+    other threats. The TG has suffered several losses at the hands of the ''xenos'', and it remains to be seen whether they can take them and the ICC on or not.<br> \
```

Comment:
You're missing an ending quotation mark on this line. it's causing the rest of this code to be 'off'

In the VSC editor, all of this should be brown text.

Reply:
I've fixed the issues and moved all of the code to misc_codex, since faction_codex is giving BYOND a headache when it comes to displaying. Will be sending screenshots shortly.

Reply:
https://i.imgur.com/3epzv8k.png
https://i.imgur.com/9LJjKfY.png
https://i.imgur.com/HwHxrk8.png
https://i.imgur.com/EzR9LnX.png